runtime/major_gc.c

```diff
@@ -453,20 +603,7 @@ static void mark_slice (intnat work)
       case Subphase_mark_main: {
           /* Subphase_mark_main is done.
              Mark finalised values. */
-          CAML_EV_BEGIN(EV_MAJOR_MARK_MAIN);
```

Comment:
This has been dropped, but the corresponding `CAML_EV_END` is still there bellow.

Reply:
Re-added.